### PR TITLE
version one

### PR DESCRIPTION
**Pull request by Trixterfox1985**
Evgen's

### CODE REVIEW

**Comment by Trixterfox1985** [comment]
my scripts

File: Car.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    public int getFuel() {
+        return fuel;
+    }
+
+    public void setFuel(int fuel) {
+        this.fuel = fuel;
+    }
+
+    //public int getConsumption() {        return consumption;    }
```
Comment (oleksandr-kazimirov) [inline Car.java:59]: commented code in the program is bad practice

File: Car_Int.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package Interface;
+
+public interface Car_Int {
```
Comment (oleksandr-kazimirov) [inline Car_Int.java:3]: interface name does not follow convention

File: Car.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+Написати клас Машина, додати наступні поля: колір, марка, розхід, обʼєм пального в баці, пробіг. Написати метод: void drive(int km) — потрібно розрахувати чи вистачить пального на поїздку
+по заданим даним, якщо вистачить вивести скільки машина проїхала і загальний пробіг, якщо не вистачить — вивести порахувати і вивести скільки машина змогла проїхати і загальний пробіг,
+якщо пального не має вивести повідомлення про необхідність заправити машину. В перших 2-х випадках зменшити обʼєм пального на величину яка залежить від заданої довжини поїздки і розходу
+пального. Написати метод: void reFuel(int fuelVolume). Написати метод: void distance() який виводить данні про машину і пробіг. Можна дописати машині любі необхідні поля, а також будь-яку
+кількість конструкторів.
+*/
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Car implements Interface.Car_Int{
```
Comment (oleksandr-kazimirov) [inline Car.java:11]: the package name must be in lowercase

**Review by oleksandr-kazimirov — APPROVED** [review]

**Pull request closed without merging**